Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py

```diff
@@ -112,30 +114,37 @@ def get_data_with_mapped_label(data, labels_mapping, tag_field):
     return new_data, dict(exist_labels_counter), dict(missing_labels_counter)
 
 
-def store_model_in_demisto(model_name, model_override, X, y, confusion_matrix, threshold, y_test_true, y_test_pred,
-                           y_test_pred_prob):
-    model = demisto_ml.train_text_classifier(X, y, True)
-    model_data = demisto_ml.encode_model(model)
-    model_labels = demisto_ml.get_model_labels(model)
+def store_model_in_demisto(model_name, model_override, X, y, confusion_matrix, threshold, tokenizer, y_test_true,
+                           y_test_pred,
+                           y_test_pred_prob, target_accuracy):
+    PhishingModel = demisto_ml.PhishingModel(tokenizer)
+    PhishingModel.train(X, y, True)
+    model_labels = PhishingModel.get_model_labels()
+    model_data = demisto_ml.phishing_model_dumps(PhishingModel)
 
     res = demisto.executeCommand('createMLModel', {'modelData': model_data,
                                                    'modelName': model_name,
                                                    'modelLabels': model_labels,
-                                                   'modelOverride': model_override,
-                                                   'modelExtraInfo': {'threshold': threshold}
+                                                   'modelOverride': model_override
+                                                   # 'modelExtraInfo': {'threshold': threshold}
                                                    })
     if is_error(res):
         return_error(get_error(res))
     confusion_matrix_no_all = {k: v for k, v in confusion_matrix.items() if k != 'All'}
     confusion_matrix_no_all = {k: {sub_k: sub_v for sub_k, sub_v in v.items() if sub_k != 'All'}
                                for k, v in confusion_matrix_no_all.items()}
+
+    y_test_pred_prob = [float(x) for x in y_test_pred_prob]
     res = demisto.executeCommand('evaluateMLModel',
                                  {'modelConfusionMatrix': confusion_matrix_no_all,
```

Comment:
we need to change the confusion matrix to be confusion matrix with considering the threshold. ask @lperryp about it

Reply:
@yguezpa plz work with @lperryp to change the confusion matrix to be the one with the threshold considiration